### DISCUSSION TITLE
Up to how many instance images, should i keep the text encoder steps below 450?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1751] [Q&A] [answered]
Do I need to increase this if i have like 500 concept images?

**Comment by TheLastBen** [comment, marked as answer]
don't use concept images, keep the text encoder below 450 and use 10 instance images

**Reply by gilroff** [reply]
ohh I thought concept images were useful if we are not training faces. has it been changed?

**Reply by TheLastBen** [reply]
they can help, but only after you evaluate the results without them

**Reply by gilroff** [reply]
ok will try
